resolvers/hospital.js

```diff
@@ -54,29 +54,27 @@ const hospitalResolvers = {
         },
       });
 
-      // Restoring associated event if event is also avaliable
+      // Restoring event association if event is also avaliable
       const associatedEvents = await db.eventHospitals.findAll({
         where: {
           hospitalId: args.id,
         },
+        include: [
+          {
+            model: db.event,
+            required: true,
+          },
+        ],
```

Comment:
do we need to explicitly require the `event` model here? i thought `eventHospitals` would already provide the event ID and hospital ID for free?

Reply:
No we need to specify it in this manner to instruct Sequelize to perform an inner join. Otherwise, it will just give us the `eventHospitals` without joins.